src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -55,6 +55,16 @@ foam.CLASS({
       name: 'browseTitle',
       expression: function(of) { return foam.String.pluralize(of.model_.label); }
     },
+    {
+      class: 'StringArray',
+      name: 'defaultColumns',
+      factory: null,
+      expression: function(of) { 
+        return of.model_.tableColumns.length 
+                ? of.model_.tableColumns
+                : of.model_.properties;
```

Comment:
Sorry I may have mislead you on this a little cuz I think I showed you the model_ approach but I think we're better off doing:
```
var tableColumns = of.getAxiomByName('tableColumns');
return tableColumns ? tableColumns.columns :
  of.getAxiomsByClass(foam.core.Property).map(p => p.name);
```
Reason being, we'll get any inherited properties or table columns this way.

Reply:
Done.